app/src/main/java/com/project/drinkly/util/MainUtil.kt

```
@@ -43,4 +43,15 @@ object MainUtil {
                     )
         }
     }
+
+    // 거리 반환 함수
+    fun formatDistance(distanceKm: Double): String {
+        return if (distanceKm < 1.0) {
+            val distanceMeter = (distanceKm * 1000).toInt()
+            "${distanceMeter}m"
+        } else {
+            String.format("%.1fkm", distanceKm)
```

detekt (1.23.8)

[warning] String.format("%.1fkm", distanceKm) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)